skyportal/handlers/api/source.py

```diff
@@ -707,6 +716,12 @@ class Validator(Schema):
             q = q.filter(Source.saved_at <= saved_before)
         if saved_after:
             q = q.filter(Source.saved_at >= saved_after)
+        if list_name:
+            q = q.join(Listing)
```

Comment:
I don't think you can add a join after applying filters -- I assume this is working in testing because none of the previous filter statements are in effect. This will need to be moved, as I don't think it will work at all as is when any of the previous filter blocks are being used

Reply:
I moved the join up to where the classifications/taxonomies are joined. But there is still a call to filter on this query above these statements, so I am not sure what the difference is.